zebra-network/src/peer/client/tests/vectors.rs

```diff
@@ -1,233 +1,102 @@
 //! Fixed peer [`Client`] test vectors.
```

Comment:
I don't understand. The coverage should be exactly the same. The `MockedClientHandle` is just a holder of all the endpoints to channels sent into the `Client` instance.

I only call it a mocked client because there is no actual outside connection.

Reply:
Oops, this was my mistake.

I opened PR #3246 so we get coverage comments on PRs.

Can you please improve the documentation for `MockedClientHandle`? And maybe change the name?

Specifically, this is unclear:
> A handle to a mocked [`Client`] instance.

I thought it meant that we'd replaced the functionality of the `Client` struct with a test-only implementation. But we're actually only mocking the channels connected to it. So maybe it's a `ClientWithMockedChannels` or `ClientTestHarness`?

Reply:
Done:

- `MockClientBuilder` -> `ClientTestHarnessBuilder`
- `MockedClientHandle` -> `ClientTestHarness`

Let me know what you think.